website/client/src/components/ui/equipBadge.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <span
+    v-if="show"
+    class="badge badge-round badge-item badge-equip"
+    :class="{'is-equipped': equipped === true}"
+    @click.stop="click"
+  >
+    <div
+      class="svg-icon color equip-icon"
+      v-html="icons.equip"
+    >
+    </div>
+    <div
```

Comment:
v-once

Reply:
is it expected to have an unequip icon in this file?

Reply:
yes, if it is equipped we need the unequipped icon on the hover to show "if you click you'll unequip it" :)